app/views/welcome/index.html.erb

```diff
@@ -1,49 +1,120 @@
+      <% @npc.stats.skills.proficiencies.each do |prof| %>
+          <li> <%= prof %></li>
+      <% end %>
+    <p>Equipment:</p>
+      <li>Armor: <%= @npc.proficiencies.armor %></li>
+      <li>Weapons: <%= @npc.proficiencies.weapons %></li>
+      <li>Tools: <%= @npc.proficiencies.tools %>, <%= @npc.background.tool_proficiency_one %>, <%= @npc.background.tool_proficiency_two %></li>
+    <p>Tools: </p>
 
   <h3>Stats:</h3>
-  <p>Passive Perception: <%= @npc.stats.passive_perception %></p>
-  <p>Core Stats:</p>
-      <li>Strength: <%= @npc.stats.core_stats.stats[:str] %></li>
-      <li>Dexterity: <%= @npc.stats.core_stats.stats[:dex] %></li>
-      <li>Constitution: <%= @npc.stats.core_stats.stats[:con] %></li>
-      <li>Intelligence: <%= @npc.stats.core_stats.stats[:int] %></li>
-      <li>Wisdom: <%= @npc.stats.core_stats.stats[:wis] %></li>
-      <li>Charisma: <%= @npc.stats.core_stats.stats[:cha] %></li>
-  <p>Saving Throws:</p>
+    <p>Hit Points: <%= @npc.stats.hit_points %></p>
+    <p>Passive Perception: <%= @npc.stats.passive_perception %></p>
+    <p>Core Stats:</p>
+      <p>Scores:</p>
+        <li>Strength: <%= @npc.stats.core_stats.stats[:scores][:str] %></li>
+        <li>Dexterity: <%= @npc.stats.core_stats.stats[:scores][:dex] %></li>
+        <li>Constitution: <%= @npc.stats.core_stats.stats[:scores][:con] %></li>
+        <li>Intelligence: <%= @npc.stats.core_stats.stats[:scores][:int] %></li>
+        <li>Wisdom: <%= @npc.stats.core_stats.stats[:scores][:wis] %></li>
+        <li>Charisma: <%= @npc.stats.core_stats.stats[:scores][:cha] %></li>
+      <p>Modifiers:</p>
+        <li>Strength: <%= @npc.stats.core_stats.stats[:modifiers][:str_mod] %></li>
+        <li>Dexterity: <%= @npc.stats.core_stats.stats[:modifiers][:dex_mod] %></li>
+        <li>Constitution: <%= @npc.stats.core_stats.stats[:modifiers][:con_mod] %></li>
+        <li>Intelligence: <%= @npc.stats.core_stats.stats[:modifiers][:int_mod] %></li>
+        <li>Wisdom: <%= @npc.stats.core_stats.stats[:modifiers][:wis_mod] %></li>
+        <li>Charisma: <%= @npc.stats.core_stats.stats[:modifiers][:cha_mod] %></li>
+    <p>Saving Throws:</p>
       <li>Strength: <%= @npc.stats.saving_throws.throws[:str] %></li>
       <li>Dexterity: <%= @npc.stats.saving_throws.throws[:dex] %></li>
       <li>Constitution: <%= @npc.stats.saving_throws.throws[:con] %></li>
       <li>Intelligence: <%= @npc.stats.saving_throws.throws[:int] %></li>
       <li>Wisdom: <%= @npc.stats.saving_throws.throws[:wis] %></li>
       <li>Charisma: <%= @npc.stats.saving_throws.throws[:cha] %></li>
-  <p>Proficiencies:</p>
-    <% @npc.stats.skills.proficiencies.each do |prof| %>
-        <li> <%= prof %></li>
-    <% end %>
-  <p>Skills:</p>
-
+    <p>Skills:</p>
       <li>Acrobatics: <%= @npc.stats.skills.skills[:acrobatics] %></li>
       <li>Animal Handling: <%= @npc.stats.skills.skills[:animal_handling] %></li>
       <li>Arcana: <%= @npc.stats.skills.skills[:arcana] %></li>
```

Comment:
Obviously a lot has been added to this, but it really is just to showcase what can be shown.